webapp-django/crashstats/supersearch/models.py

```diff
@@ -74,24 +83,42 @@ def __init__(self):
             and x['is_mandatory']
         )
 
+        # Add histogram fields for all 'date' or 'number' fields.
+        histogram_fields = []
+        for x in all_fields.values():
```

Comment:
Not important but let's avoid single character variables unless it's a list comprehension or something like `for i in range(number):`